test/common_utils.py

```diff
@@ -559,6 +563,42 @@ def __exit__(self, exec_type, exec_value, traceback):
 except ImportError:
     print('Fail to import hypothesis in common_utils, tests are not derandomized')
 
+disabled_test_from_issues = None
+disabled_query = 'is%3Aissue+is%3Aopen+label%3A%22topic%3A+flaky-tests%22+repo:pytorch/pytorch+in%3Atitle+DISABLED'
+def check_disabled(test_name):
+    global disabled_test_from_issues
+    if disabled_test_from_issues is None:
+        disabled_test_from_issues = set()
+
+        def read_and_process():
+            with open('disabled_tests.json', 'r') as opened:
+                contents = opened.read()
+            the_response = json.loads(contents)
+            for item in the_response['items']:
+                title = item['title']
+                key = 'DISABLED '
+                if title.startswith(key):
+                    test_name = title[len(key):].strip()
+                    disabled_test_from_issues.add(test_name)
+
+        if not IS_SANDCASTLE:
+            try:
+                try:
+                    read_and_process()
+                except FileNotFoundError:
+                    repo_dir = tempfile.mkdtemp()
+                    check_call(['git', 'clone', 'https://github.com/zdevito/pytorch_disabled_tests.git', repo_dir])
```

Comment:
I don't think there's any reason to clone the whole repo. Just fetch the file from https://raw.githubusercontent.com/zdevito/pytorch_disabled_tests/master/result.json